docs/usage.md

```diff
@@ -111,7 +111,7 @@ tar xvzf db.tar.gz
 And then passed to the pipeline with:
 
 ```bash
---annotation_bakta_db /<path>/<to>/db/
+--annotation_bakta_db_localpath /<path>/<to>/db/
```

Comment:
it sounds like this option is oriented towards HPC or laptop users, but will it also work for cloud storage (AWS S3, GCP CS, etc.)?

Reply:
@jfy133 expertise? Would something like `bakta --db [path of S3 bucket]` work? In the `funcscan.nf` file, we pass a channel with the path to the module.

Reply:
I _think_ it should work as I believe any `*Path` Nextflow function will stage accordingly, but I think it would still be _safer_ to load this with `file()` and pass it in like that - because that way it will definitely support resolving both filesystem paths and URLs.

Also @jasmezz you didn't add this path to `checkPathParamList` on line 13 of funcscan.nf (sorry my oversight while reviewing).

Reply:
Huh, yes it is present in line 13 of funcscan.nf! :eyes:

Reply:
Let's optimize how we provide paths for the next release. It should also affect other parameters than this one. (The next tools release is just around the corner, so...)